pkg/operator/operators/klusterlet/controllers/klusterletcontroller/klusterlet_managed_reconcile.go

```diff
@@ -38,7 +38,7 @@ var (
 		"klusterlet/managed/klusterlet-work-clusterrole.yaml",
 		"klusterlet/managed/klusterlet-work-clusterrole-execution.yaml",
 		"klusterlet/managed/klusterlet-work-clusterrolebinding.yaml",
-		"klusterlet/managed/klusterlet-work-clusterrolebinding-execution.yaml",
+		"klusterlet/managed/klusterlet-work-clusterrolebinding-aggregate.yaml",
```

Comment:
should we also add the old `open-cluster-management:{{ .KlusterletName }}-work:execution` in the list?

Reply:
defined another cleanedManagedStaticResourceFiles to delete all manifests.